frontends/web/src/routes/buy/guide.tsx

```diff
@@ -36,9 +36,13 @@ export default function BuyGuide({ name }: BuyGuideProps) {
       }} shown={true} />
       <Entry key="guide.buy.protection" entry={{
         link: {
-          text: t('buy.info.disclaimer.privacyPolicy'),
+          text: t('buy.info.disclaimer.moonpayPrivacyPolicy'),
           url: 'https://www.moonpay.com/privacy_policy',
         },
+        link2: {
+          text: t('buy.pocket.data.link'),
+          url: 'https://pocketbitcoin.com/policy/privacy',
```

Comment:
nit: could add https://pocketbitcoin.com/de/policy/privacy and https://pocketbitcoin.com/it/policy/privacy (German is fine but Italian seems to not be translated yet).

moonpay is also hardcoded to EN, so feel free to keep as is.

Reply:
I think it is better to keep this PR simple and make a separated PR where we call localized policies for both moonpay and pocket. wdyt?